### ISSUE TITLE
Suggest not using 'Assert.Multiple' with only one assertion or when empty.

### ISSUE THREAD

**Issue by brian-reichle** [nunit/nunit.analyzers#473]
There isn't any point using `Assert.Multiple` if it contains zero or one assertions, so should produce a diagnostic suggesting that it be removed.

```csharp
Assert.Multiple(() => // <-- should warn that this Assert.Multiple call only contains a single assertion.
{
    Assert.That(foo, Is.EqualTo("bar"));
});

Assert.Multiple(() =>  // <-- should warn that this Assert.Multiple call is empty.
{
});
```

The code fix should simply replace the first call with the contained assertion, and delete the second call.

```csharp
Assert.That(foo, Is.EqualTo("bar"));
```

**Comment by manfred-brands** [comment]
The idea is good, it is just not that straightforward. Mainly because `Assert` could be called indirectly through other methods, either local or external. See also #112.
Like in #112 the last case can be detected if the body is empty, but if it contains just comments, like temporary commented out code you would loose all those.

**Comment by brian-reichle** [comment]
Commented out code is a pet-hate of mine :), but the indirect assertions is a good point.
